src/core/eme/handle_session_events.ts

```diff
@@ -151,6 +151,7 @@ export default function handleSessionEvents(
         .pipe(
           catchError((error: Error) => {
             throw new EncryptedMediaError("KEY_STATUS_CHANGE_ERROR", error, true);
+            return EMPTY;
```

Comment:
unreachable

Reply:
tslint had some issues without returning an empty observable, even if unreachable ...
Seems to be fixed